src/backend/data/post.js

```diff
@@ -148,7 +149,16 @@ class Post {
       article.date = article.pubdate;
     }
 
+    // All the Youtube feed return an array off html
+    if (determinePostType(article.link) === 'video' && Array.isArray(article.content)) {
+      article.content = article.content.join(' ');
+    }
+
+    // Wrap an <a> tag on any link inside our content
+    article.content = linkifyHtml(article.content);
```

Comment:
This needs tests.

It also needs to be done in the Posts service at the same time.

Reply:
@humphd I talked to @manekenpix  and I belive that `Post` service is for fetching the data from `redis` to other services that needed the data. So I did not think we need to `linkify` it as the change work ? For the test part would you want me to file the issue or to test it as well in this PR ?

Reply:
I'm sorry, I meant parser service, not posts, you're right.

For tests, let's add them in this PR so we don't break anything.  Add more tests under https://github.com/Seneca-CDOT/telescope/blob/master/test/post.test.js#L185 to pass in content that has an array, that has text vs. html, has links and expect them to be linkified, etc.  We don't need a lot, just some coverage of this code change.

Reply:
I just want to understand more . I guess the change for the parser service will serve our way on moving away from `backend` in the future ? Because when I talk to people on Slack I belive the `parser` service is still under construction.

Reply:
This part of the backend also lives in the parser service, so if you add this here, you need to add it to the parser service (eventually) too.

"When I talk to people on Slack..." where are you talking to people?  I'm on Slack and see none of this.  "Open source" means not talking in DMs.

Reply:
I think we can merge this first (once it's ready) and then @TueeNguyen can add these changes to the `parser` service, since he's working on it?

Reply:
Agree.  But let's include tests before this goes in.